daprdocs/content/en/operations/components/setup-state-store/supported-state-stores/_index.md

```diff
@@ -25,6 +25,12 @@ no_list: true
 | RethinkDB                                                      | ✅   | ✅                  | ✅    | Alpha  |
 | [Zookeeper]({{< ref setup-zookeeper.md >}})                    | ✅   | ❌                  | ✅    | Alpha  |
 
+
```

Comment:
@trondhindenes one more change. repeated in line 48.

Reply:
I fixed that